src/controllers/dashboard/plugins/installed/index.js

```diff
@@ -59,7 +79,7 @@ function getPluginCardHtml(plugin, pluginConfigurationPages) {
 
     html += "<div class='cardText'>";
     html += configPage && configPage.DisplayName ? configPage.DisplayName : plugin.Name;
-    html += '</div>';
+    html += '<br/>' + globalize.translate('LabelStatus') + ` ${plugin.Status}</div>`;
```

Comment:
```suggestion
    html += `<br/>${globalize.translate('LabelStatus')}${plugin.Status}</div>`;
```

Not sure what are you trying to do here is correct though

Reply:
@ferferga you missed the space between `LabelStatus` and `Status`

Reply:
@dmitrylyzo Oh I see

Edited